src/version.cmake

```diff
@@ -0,0 +1,6 @@
+EXECUTE_PROCESS(
```

Comment:
Hm. I am using CMake 3.1.0 and it seems like it works fine as it is:


```
[ himera ] brezerk@pts/4:808  ~/develop/Cataclysm-DDA-Fork/build $
echo "lol" > ../src/version.h         
[ himera ] brezerk@pts/4:809  ~/develop/Cataclysm-DDA-Fork/build $
cmake ..   
...
-- Configuring done
-- Generating done
-- Build files have been written to: /home/brezerk/develop/Cataclysm-DDA-Fork/build
[ himera ] brezerk@pts/4:810  ~/develop/Cataclysm-DDA-Fork/build $
make
[  2%] Built target chkjson
-- Found Git: /usr/bin/git (found version "2.3.5")
[  2%] Built target get_version
[  2%] Built target lua_bindings
...
[ himera ] brezerk@pts/4:811  ~/develop/Cataclysm-DDA-Fork/build $
cat ../src/version.h
#define VERSION "0.9-19330-gb56c7af"
[ himera ] brezerk@pts/4:813  ~/develop/Cataclysm-DDA-Fork/build $
git describe --tags --always --dirty --match "[0-9]*.[0-9]*"
0.9-19330-gb56c7af
```

are you still having problems with current cmake version?